<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgAddNew</class>
 <widget class="QDialog" name="dlgAddNew">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>318</width>
    <height>180</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Item</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>88</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAdd">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>140</y>
     <width>88</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spbQuantity">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>80</y>
     <width>71</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtId">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>30</y>
     <width>141</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>58</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Item Id:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>71</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Quantity:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>dlgAddNew</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>54</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>182</x>
     <y>157</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
